cmd/lakectl/cmd/log.go

```diff
@@ -26,17 +30,27 @@ Metadata:
 {{ end }}{{ if .Pagination  }}
 {{.Pagination | paginate }}{{ end }}`
 
+// escapeDot escapes a string to match the graphviz dot-graph syntax
+// based on: https://graphviz.org/doc/info/lang.html
+func escapeDot(s string) string {
+	var buf bytes.Buffer
+	_ = xml.EscapeText(&buf, []byte(s))
+	return buf.String()
+}
```

Comment:
Can we use html.EscapeString instead? from the docs it looks like the labels as html encoded.

Reply:
Done